src/settings.py

```diff
@@ -62,7 +62,7 @@ class Settings(BaseSettings):
     VALIDATE_CERTS: bool = True
 
     # Адреса электронной почты администраторов
-    EMAIL_ADMIN: list[EmailStr] = []
+    EMAIL_ADMIN: EmailStr
```

Comment:
Так работать будет?)
```suggestion
    EMAIL_ADMIN: list[EmailStr] | EmailStr = []
```

Reply:
EMAIL_ADMIN: list[EmailStr] | EmailStr = [] - я так пробовал, у меня не работало. Вечером посмотрю и попробую еще не сколько вариантов)

Reply:
Похоже теперь норм должно работать) Обновил)